.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -0,0 +1,109 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+
+schedules:
+- cron: "0 0 * * 0"
+  displayName: Weekly Build
```

Comment:
>Weekly [](start = 15, length = 6)

why weekly? I guess you need at least daily to test it for a while.

Reply:
There are not much change, it will consume 8 agents * 10 hours (average build time) in 4 cores. We can trigger it on demand in the Azure Pipelines.

Reply:
Changed to daily build.